pyiron_atomistics/atomistics/master/phonopy.py

```diff
@@ -510,4 +522,11 @@ def validate_ready_to_run(self):
             raise ValueError("Phonopy reference jobs should be static calculations, but got {}".format(
                 self.ref_job._generic_input["calc_mode"]
             ))
+        if self.input["number_of_snapshots"] is not None:
+            with ImportAlarm(
+                    "Phonopy with random snapshot displacement relies on the alm module but this unavailable. Please "
+                    "ensure your python environment contains alm, e.g. by running `conda install -c conda-forge alm`."
+                    "Note: at time of writing alm is only available for Linux and OSX systems."
+            ) as import_alarm:
```

Comment:
That doesn't quite work.  `import_alarm` is designed to be a decorator after the `with` statement.  I made a quick [patch](https://github.com/pyiron/pyiron_base/pull/328) to trigger the warning manually.

Reply:
Pushed an update -- like this?

Reply:
No, the message is updated after the code exits the with statement.  I'll update the docstring in the other PR.